### PR TITLE
build: remove symlink to `/usr/local/bin/2to3` on buildscript

### PR DESCRIPTION
**Pull request by reidzeibel**
another attempt to fix build issue, similar to https://github.com/AtB-AS/mittatb-app/pull/4366

### CODE REVIEW

File: .github/workflows/build-store-ios.yml [hunk 1]
```diff
@@ -37,7 +37,9 @@ jobs:
         with:
           ssh-private-key: ${{ secrets.MATCH_SSH_PRIVATE_KEY }}
       - name: Generate native assets
-        run: brew install --overwrite imagemagick && yarn generate-native-assets
+        run: |
+          rm '/usr/local/bin/2to3'
+          brew install imagemagick && yarn generate-native-assets
```
Comment (rosvik) [inline .github/workflows/build-store-ios.yml:42]:
```suggestion
          rm '/usr/local/bin/2to3'
          brew install imagemagick
          yarn generate-native-assets
```

**Review by rosvik — APPROVED** [review]
Worth a shot 🤞 Nit: Might as well use newlines for all commands when we do it for the one you added.

**Review by rosvik — CHANGES_REQUESTED** [review]
I'll trigger a staging build, and we'll see it that works before we merge this

File: .github/workflows/build-store-ios.yml
Comment (rosvik) [inline .github/workflows/build-store-ios.yml:1]: @reidzeibel I'm realising we're running the build with the other yml file from the one we're changing 😅 Try making (some of) these changes to build-staging-ios.yml instead.

**Comment by reidzeibel** [comment]
Build success on staging : https://github.com/AtB-AS/mittatb-app/actions/runs/8141619752 

ready for review @rosvik

**Review by rosvik — APPROVED** [review]
Great job figuring this out!

This doesn't look like a very robust solution (It will probably break the moment imagemagick updates the python version they rely on? 😅), so after this, I think we should use some time to find a better solution.

But for now, let's hope this get the build up and running again 🚀

**Pull request merged**